tf2onnx/tfonnx.py

```diff
@@ -31,7 +32,17 @@
 TARGET_RS4 = "rs4"
 TARGET_CAFFE2 = "caffe2"
 POSSIBLE_TARGETS = [TARGET_RS4, TARGET_CAFFE2]
-DEFAULT_TARGET = [TARGET_RS4, TARGET_CAFFE2]
+DEFAULT_TARGET = []
+PREFERRED_OPSET = 7
+
+
+def find_opset(opset):
+    if opset is None or opset == 0:
+        opset = defs.onnx_opset_version()
+        if opset > PREFERRED_OPSET:
```

Comment:
does this mean, though user passed --opset 8, 7 is still be used during conversion? is this expected?

Reply:
sorry just mis-understood. by default, 7 is used.